src/pages/components/form/usage.mdx

```diff
@@ -117,215 +115,378 @@ do you gain by collecting this information?
 
 ## Formatting
 
-All forms are comprised of six elements:
-
-- **Labels:** Inform users what the corresponding input fields mean.
-- **Input fields:** Enable users to provide information. Information can be
-  entered through a variety of different input fields ranging from text fields,
-  checkboxes, and many other types.
-- **Helper text:** Provides assistance on how to fill out a field. Helper text
-  is optional.
-- **Placeholder text:** Hints at what goes into a field. Placeholder text is
-  optional.
-- **Actions:** Allow users to submit a form.
-- **Validation:** Ensures the data submitted by the user conforms to acceptable
-  parameters.
+### Anatomy
 
 <Row>
 <Column colLg={8}>
 
-![effective form design](images/form-usage-1.png)
+<Tabs>
 
-</Column>
-</Row>
+<Tab label="Fixed">
 
-### Button placement
+![Anatomy of a fixed form](images/form-usage-1.png)
 
-On non-modal or in-page forms, align single buttons or Secondary / Primary
-button groups to the left. Buttons should align with the form controls
-regardless of the user‘s window width.
+</Tab>
 
-<Row>
-<Column colLg={8}>
+<Tab label="Fluid">
+
+![Anatomy of a fluid form](images/form-usage-1-fluid.png)
```

Comment:
Anatomy image (fluid):
The spacing between the text and the icon should be 8px, but I think it's at 16px? 
<img width="443" alt="Screen Shot 2023-01-24 at 1 05 41 PM" src="https://user-images.githubusercontent.com/43969356/214387871-195d6967-1b25-44ca-adf7-85f76eeb621b.png">